core/cloudflow-operator/src/main/scala/cloudflow/operator/action/Action.scala

```diff
@@ -65,24 +65,23 @@ object Action {
   val ConflictCode = 409
 
   /**
-   * Creates a [[CreateAction]].
+   * Creates a [[CreateOrUpdateAction]].
    */
-  def create[T <: ObjectResource](resource: T, editor: ObjectEditor[T])(implicit format: Format[T],
-                                                                        resourceDefinition: ResourceDefinition[T]) =
-    new CreateAction(resource, format, resourceDefinition, editor)
+  def createOrUpdate[T <: ObjectResource](resource: T, editor: ObjectEditor[T])(implicit format: Format[T],
```

Comment:
create and update shared where exactly the same, so made the name more explicit and removed duplication.